src/components/Header.js

```diff
@@ -1 +1,28 @@
-// Header
\ No newline at end of file
+import React from 'react';
+import Button from './Button';
+import data from '../constants/data';
```

Comment:
You dont need this import, just pass the data to header component has a props using the ColorSyncApp component